databook/physiology/ephys/np-ultra/npultra-psychedelics-accessing-data.md

```diff
@@ -0,0 +1,112 @@
+| pre_peak_trough_ratio    | he absolute value of the ratio of the amplitudes found at the spike trough and pre-trough peak voltage in the average spike waveform.     |
+| repolarization_slope    | The slope of the voltage from the trough to the peak.     |
+| recovery_slope    | The slope of the voltage from the peak back to baseline (typically 0 $\mu$V).     |
+| avg_ISI    | The average interspike interval (in seconds) taken across all spike, excluding those during the optotagging epochs.     |
+| baseline_FR    | the average firing rate of spikes taken in the 200 ms window before each optotagging stimulus (used for optotagged neuron identification).     |
+| evoked_FR    | the average firing rate of spikes taken in the optotagging stimulus window duration (used for optotagged neuron identification).     |
+| footprint    | The average radius (in $\mu$m) of the spike waveform across the surface the probe before attenuating into noise.    |
+| layer    | The cortical layer a given unit was found in.     |
+| region    | The cortical area a given unit was found in.     |
+| optotagged    | Whether a unit was found to reliably be excited by a photostimulus. 0 for not optotagged, 1 for optotagged.     |
+| Cell type    | The putative cell type for a give unit. Can be "Sim1", "Tlx3", 'RS', 'FSl', or 'FSs'.     |
+| spread    | The vertical spread (in $\mu$m) of a spike waveform along the column of electrodes containing peak amplitude channel.     |
+| bAP_extent    | The distance traveled (in $\mu$m) of a detected back-propagating action potential in the average spike waveform.     |
+| bAP_attenuation    | The proportion of the maximum amplitude spike that a bAP attenuates by before falling below noise (~10 $\mu$V).     |
+| bAP_idx    | The product of the bAP extent and one minus the bAP attenuation.     |
+| burst_idx    | The ratio of the number of detected burst events to the total number of spikes.   |
+| num_bursts    | The total number of detected burst events.     |
+| burst_dur    | The average duration of a burst (in ms)     |
+| burst_proportion    | The proportion of spikes allocated to burst events.     |
+
+## Experimental epochs
+You may wish to know the time points at which different parts of the experimental session took place. For instance, you may wish to know when the pre- and post-injection spontaneous activity epochs took place, you can call the epoch table as:
+
+```{code-cell} ipython3
+# get the different epochs and their beginning and end times
+epochs = nwbfile_zarr.stimulus['epochs'].to_dataframe()
+```
+
+## Stimulus data
+The only timing-based visual stimulation used during these experiments were Gabor patches presented at various orientations and locations on the screen during receptive field mapping epochs. To find trial-specific information for visual stimuli:
+
+```{code-cell} ipython3
+# load the stimulus table
+stimulus_table = nwbfile_zarr.stimulus['visualstim'].to_dataframe()
+```
+
```

Comment:
Can you add a list of the what these parameters for this are and define them? Likewise for the optotagging